applications/luci-app-spoofer/Makefile

```diff
@@ -0,0 +1,18 @@
+#
+# Copyright (C) 2018 The Regents of the University of California
```

Comment:
Do you actually have a copyright assignment on file with UC?

Reply:
Yes, I do.  (I'm the same person who added the spoofer package to openwrt, and released the upstream spoofer package it's built on.)